migrate/src/main/scala/migrate/MigrateLib.scala

```diff
@@ -0,0 +1,142 @@
+package migrate
+
+import scala.util.Try
+
+import migrate.Lib213.macroLibs
+import migrate.MigrateLib._
+import migrate.interfaces.Lib
+import migrate.utils.CoursierHelper
+
+sealed trait MigrateLib extends Lib {
```

Comment:
I think that `MigrateLib` is too vague. I would use `LibToMigrate` or `InputLib` and then `MigratedLib` or `OutputLib`